mne/io/open.py

```diff
@@ -255,7 +257,6 @@ def _show_tree(fid, tree, indent, level, read_limit, max_str):
                 out += [next_idt + prepend + str(k) + ' = ' +
                         '/'.join(this_type) + ' (' + str(size) + ')' +
                         postpend]
-                out[-1] = out[-1].replace('\n', u'¶')
```

Comment:
Though that itself contradicts with the signature where you can specify your output to be "str".